src/ValueConverter/MappingValueConverter.php

```diff
@@ -24,7 +24,7 @@ public function __construct(array $mapping)
 
     public function __invoke($input)
     {
-        if (!isset($this->mapping[$input])) {
+        if (!\array_key_exists($input, $this->mapping)) {
```

Comment:
`array_key_exists` is slower than `isset`, so can you change this to:


```php
if (isset(…) || array_key_exists(…)) {
    return …
}

throw …
```

Reply:
hah, Premature Optimization?

You're the boss!

Reply:
it is done